package-lock.json

```diff
@@ -33842,7 +33842,7 @@
         "eslint-plugin-import": "^2.30.0",
         "eslint-plugin-jsx-a11y": "^6.10.0",
         "eslint-plugin-react": "^7.36.1",
-        "eslint-plugin-react-hooks": "*",
+        "eslint-plugin-react-hooks": "beta",
```

Comment:
is this change intended?

Reply:
i did an `npm i` and this got updated - i'll change it back in the interest of safety if you're concerned!